templates/valkey-try-me.html

```diff
@@ -0,0 +1,277 @@
+{% extends "fullwidth.html" %}
+{%- block head -%}
+    <title>Try Valkey</title>
+
+    <!-- Scripts -->
+    <script src="https://dtq9kvai1345o.cloudfront.net/vos/v86/libv86.js"></script>
+    <script src="https://dtq9kvai1345o.cloudfront.net/vos/xterm/xterm.min.js"></script>
+    <script src="https://dtq9kvai1345o.cloudfront.net/vos/pako/pako.min.js"></script>
+    <script src="https://dtq9kvai1345o.cloudfront.net/vos/v86/serial_xterm.js"></script>
+    
+    <!-- Styles -->
+    <link rel="stylesheet" href="https://dtq9kvai1345o.cloudfront.net/vos/xterm/xterm.css" />
```

Comment:
Can we host this just on the website?

Reply:
I think that would drive some features to be implemented cross repositories and we should avoid that. I agree the fact that the css files are more visualisation related might seem rational to include in the webpage repo, but since they are somewhat correlated with the v86 objects I would suggest to avoid it at this point.